README.md

```diff
@@ -44,3 +44,23 @@ It is possible to use a `substra-backend` skaffold file as the source for the ne
 ```
 SUBSTRA_TESTS_SKAFFOLD_FILEPATH=$SUBSTRA_SOURCE/substra-backend/skaffold.yaml make test
 ```
+
+# Test design guidelines
+
+When adding or modifying tests, please follow these guidelines.
+
+The tests suite:
+- Must be easy to configure through settings files to define the substra network to target
+- Must be independent from the substra network
+  - Substra network must be started separately
+  - Substra network can be located locally / on the cloud
+  - Substra network can be started through docker-compose or through kubernetes
+- Requires a setup with at least 2 organisations
+- Should be able to run multiple times without restarting the network
+
+Each new test:
+- Should have a deterministic behavior (not fail randomly)
+- Must be fast to run:
+  - avoid adding latency
+  - group related long running tests
+- Should not complete before all the tuples it created have been executed: it is required to ensure that the next test will be launched with a susbtra network ready to execute new tuples
```

Comment:
- What do you think of breaking down this sentence into two sentences (in general, it tends to read better)
- "it is required to ensure" => "This requirement ensures" ?

Reply:
Yes, it's better.